### DISCUSSION TITLE
Incorrect documentation (?)

### DISCUSSION THREAD

**Discussion by dylanklohr** [vercel/turborepo#3195] [Help] [answered]
I'm refamiliarizing myself with workspace-specific tasks this morning and noticed a potential conflict in the [documentation](https://turbo.build/repo/docs/core-concepts/monorepos/running-tasks#specific-workspace-tasks) that I wanted to verify prior to submitting a fix. 

In the notes for workspace specific tasks above, it notes 

> This explicit configuration for frontend#deploy may seem to conflict with the `test` and `deploy` task configurations, but it does not. Since test and deploy do not have dependencies on other workspaces (e.g. `^<task>`)...

Based on the screenshot below, I'm not sure that the `test` task being referenced does not depend on another workspace? Perhaps I'm misunderstanding?

![ScreenShot2023-01-06 at 08 05 22@2x](https://user-images.githubusercontent.com/17390173/211051860-c191c008-b7c2-4584-8678-4c1a9f8b075a.png)

Thanks for your time 🙂

**Comment by anthonyshew** [comment, marked as answer]
It looks like this page has been updated since this discussion was opened. Sorry the team wasn't able to make it here at the time!

Closing as stale.
